resources/helper/StructurePlacer.gd

```diff
@@ -0,0 +1,97 @@
+extends Node
```

Comment:
Why Node?

Reply:
Bc godot is dumb sometimes and defaulted to that again on my PC